### PR TITLE
Changed offline activity list to use links [#177363830]

### PR DESCRIPTION
**Pull request by dougmartin**
This will get us a "clean slate" for the new activity

### CODE REVIEW

**Comment by dougmartin** [comment]
NOTE: the reduction in test coverage is because we still have the Cypress tests disabled for offline mode.

File: src/components/offline-activities.tsx [hunk 1]
```diff
@@ -1,26 +1,23 @@
...
-  const {name, manifestName} = props.activity;
+const OfflineActivityListRow = (props: {activity: OfflineActivity}) => {
+  const {name, manifestName, contentUrl} = props.activity;
   const displayName = manifestName.length > 0 ? `${manifestName}: ${name}` : name;
-  const handleSelectActivity = () => props.onSelectOfflineActivity(props.activity);
   return (
     <tr>
       <td>TDB</td>
-      <td onClick={handleSelectActivity} className="activity">{displayName}</td>
+      <td className="activity"><a href={`?activity=${encodeURIComponent(contentUrl)}`}>{displayName}</a></td>
```
Comment (scytacki) [inline src/components/offline-activities.tsx:14]: I think for this to work right you will need to set `activity=${resourceUrl}&contentUrl=${contentUrl}`

**Review by scytacki — CHANGES_REQUESTED** [review]
Need to include activity param and contentUrl param.

**Comment by dougmartin** [comment]
@scytacki I added the contentUrl to the query params here: https://github.com/concord-consortium/activity-player/pull/185/commits/3490041c08321186a853a16139e61a2321aae66c

**Review by scytacki — APPROVED** [review]
looks good

**Pull request merged**